live-build/misc/live-build-hooks/90-raw-disk-image.binary

```diff
@@ -114,6 +114,15 @@ zfs mount rpool/ROOT/ubuntu
 #
 rsync --info=stats3 -Wa binary/* /mnt/
 
+#
+# Set up a swap device.
+#
+zfs create -V 4G -b $(getconf PAGESIZE) \
```

Comment:
These lines are failing the "shfmt" check; see [here](https://travis-ci.com/delphix/appliance-build/jobs/128450236)
```
 zfs create -V 4G -b $(getconf PAGESIZE) \
-    -o logbias=throughput -o sync=always \
-    -o primarycache=metadata rpool/swap
+	-o logbias=throughput -o sync=always \
+	-o primarycache=metadata rpool/swap
 mkswap -f /dev/zvol/rpool/swap
-echo /dev/zvol/rpool/swap none swap defaults 0 0 >> /mnt/etc/fstab
+echo /dev/zvol/rpool/swap none swap defaults 0 0 >>/mnt/etc/fstab
```

If you have a bootstrap system set up, like the README describes, you can run the check like this:
```
./scripts/docker-run make shfmtcheck
```

You can also have shfmt automatically fix the files too, with something like this:
```
shfmt -w $(find scripts -type f -executable) \
    $(find live-build/misc/live-build-hooks -type f -executable)
```

Reply:
It looks like "shellcheck" also wants some changes (see [here](https://travis-ci.com/delphix/appliance-build/jobs/128450235))
```
In live-build/misc/live-build-hooks/90-raw-disk-image.binary line 120:
zfs create -V 4G -b $(getconf PAGESIZE) \
                    ^-- SC2046: Quote this to prevent word splitting.
```